gubernator/testgrid.py

```diff
@@ -104,14 +105,17 @@ def path_to_query(path):
     options = {}
     for dashboard in get_config().get('dashboards', []):
         dashboard_name = dashboard['name'][0]
-        for tab in dashboard['dashboard_tab']:
+        tabs = dashboard['dashboard_tab']
+        for tab in tabs:
+            if 'base_options' in tab:
```

Comment:
I expect a future where all tabs define this. Can we make it prefer one without rather than require?